### PR TITLE
Lineup: Sync the theme with dotorg version

### PR DESCRIPTION
**Pull request by iamtakashi**
#### Changes proposed in this Pull Request:
During the review, I edited the theme to address the issue raised in the review without updating the theme in this repo. It was my mistake, and I will not do that going forward. This attempts to sync the theme with the one in the dotorg repository. 

Reference: https://themes.trac.wordpress.org/changeset?old_path=lineup/1.0.2&new_path=lineup/1.0.3

### CODE REVIEW

File: package-lock.json
Comment (mikachan) [inline package-lock.json:1]: These changes to the package-lock.json file should be removed from this PR - I'm guessing they were included by accident.

**Comment by iamtakashi** [comment]
Oops, sorry, @mikachan. I have now deleted the package-lock.json.

**Comment by mikachan** [comment]
Thanks @iamtakashi! It looks like the whole file may have been deleted, you just need to remove the changes to it so this PR doesn't introduce any changes 😄

**Comment by iamtakashi** [comment]
Sorry, I didn't know what I was doing 😓 

I think it's now ok, no?

**Comment by mikachan** [comment]
Yes that's perfect, thank you!

**Pull request merged**